__tests__/integration.test.js

```diff
@@ -0,0 +1,20 @@
+const request = require('supertest');
+const app = require('../app.js');
+const seed = require('../db/seeds/seed');
+const db = require('../db/connection');
+const data = require('../db/data/test-data');
+
+beforeAll(() => seed(data));
+afterAll(() => db.end());
+
+describe('selectTopics()', () => {
+  test('Selecting all Topics', () => {
+    return request(app)
+      .get('/api/topics')
+      .expect(200)
+      .then(({ body }) => {
+        // console.log(body.topics);
+        expect(body.topics).toHaveLength(3);
```

Comment:
This test isnt enough for us to be confident that we actually get back an array of topics. An assertion to check what each element in this array looks like will need to be added here as well.